readme.md

```diff
@@ -37,6 +37,9 @@ Do this every time you want to deploy, or have your CI server do it.
 5. run `./deploy.sh`
 
 ### options
+
+`deploy.sh [--config <FILE>] [<options>] [<directory> [<branch> [<repository>]]]`
```

Comment:
the flag in the script is `--config-file`, but actually, how about we change it to just `--config`?